src/actions/index.js

```diff
@@ -458,10 +501,108 @@ function updateOutput(
         });
     }
 
+    // Clientside hook
+    if (!isNil(clientside_function) && !isEmpty(clientside_function)) {
+        let returnValue;
+        try {
+            returnValue = window[clientside_function.namespace][
+                clientside_function.function_name
+            ](
+                ...(has('inputs', payload)
+                    ? pluck('value', payload.inputs)
+                    : []),
+                ...(has('state', payload) ? pluck('value', payload.state) : [])
+            );
+        } catch (e) {
+            /* eslint-disable no-console */
+            console.error(
+                `The following error occurred while executing ${
+                    clientside_function.namespace
+                }.${clientside_function.function_name} ` +
+                    `in order to update component "${payload.output}" ⋁⋁⋁`
+            );
+            console.error(e);
+            /* eslint-enable no-console */
+
+            /*
+             * Update the request queue by treating a successful clientside
+             * like a failed serverside response (500 status code)
+             */
+
+            /* eslint-disable no-magic-numbers */
+            updateRequestQueue(true, 500);
+            /* eslint-enable no-magic-numbers */
```

Comment:
I'm not quite clear what `status` gets used for in the request queue, but I'd either put `500` as `STATUS.ERROR` or, perhaps better yet, create a new value just for this purpose so we can tell the difference. We're using `status: 'loading'` in the queue initially, so we're not restricted to http codes... perhaps this can just be `status: 'clientside error'`?